spec/controllers/application_controller/timezone_spec.rb

```diff
@@ -6,15 +6,15 @@
   context "#get_timezone_offset" do
     context "for a server" do
       it "with a system default" do
-        stub_settings(:server => {:timezone => "Eastern Time (US & Canada)"})
+        stub_server_settings(:server, 'timezone' => "Eastern Time (US & Canada)")
```

Comment:
I'm pretty sure you can use `:server => {:timezone => "Eastern Time (US & Canada)"}` as original